laws/shared/src/test/scala/algebra/laws/LawTests.scala

```diff
@@ -191,5 +191,6 @@ trait LawTestsBase extends FunSuite with Discipline {
   }
 
   laws[OrderLaws, Int]("fromOrdering").check(_.order(Order.fromOrdering[Int]))
+  laws[OrderLaws, Array[Int]].check(_.order)
```

Comment:
They just check that order is consistent, not that it is dictionary ordering. I wasn't sure where to put normal non-law-tests...